Frameworks/CoreText/CTFont.mm

```diff
@@ -945,6 +945,12 @@ CFTypeID CTFontGetTypeID() {
     return __kCTFontTypeID;
 }
 
+// Private function for getting the XAML-compatible family name
+CFStringRef _CTFontGetCompatibleFamilyName(CTFontRef font) {
```

Comment:
Can't call directly from UIKit since the _DWrite version needs the _dwriteFontFace, so rename it is 🔹